fastlane/lib/fastlane/actions/create_app_on_managed_play_store.rb

```diff
@@ -85,24 +85,18 @@ def self.available_options
             short_option: "-k",
             env_name: "SUPPLY_DEVELOPER_ACCOUNT_ID",
             description: "The ID of your Google Play Console account. Can be obtained from the URL when you log in (`https://play.google.com/apps/publish/?account=...` or when you 'Obtain private app publishing rights' (https://developers.google.com/android/work/play/custom-app-api/get-started#retrieve_the_developer_account_id)",
-            optional: false,
```

Comment:
Is `optional: false` the default or why could these (this one and the others from the params) be removed?